src/hammer-tech/hammer_tech.py

```diff
@@ -712,6 +712,9 @@ def extract_tarballs(self) -> None:
         for tarball in self.config.tarballs:
             target_path = os.path.join(self.extracted_tarballs_dir, tarball.path)
             tarball_path = os.path.join(self.get_setting(tarball.base_var), tarball.path)
+            if not os.path.isfile(tarball_path):
+                raise ValueError("Path {0} does not point to a valid tarball!".format(tarball_path))
+                continue
```

Comment:
Isn't this `continue` redundant?

Reply:
True, `ValueError` handles it. It's left from when I was doing a logger error instead.